documentation/release_notes.markdown

```diff
@@ -10,6 +10,22 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.8.0
+
+Released 14 January 2020
+
+### New features
+
+- When requesting that a certificate be signed, the `certificate-status` API endpoint can now accept a TTL in its body under the key `cert_ttl`, which determines the validity period of the cert being signed. The unit defaults to seconds, but you can specify the unit. See [configuration](https://puppet.com/docs/puppet/latest/configuration.html#configuration-settings) for a list of Puppet's accepted time unit markers. [SERVER-2678](https://tickets.puppetlabs.com/browse/SERVER-2678)
+
+- This release adds a new JRuby pool architecture that maintains a single JRuby instance through which requests to Puppet Server are run concurrently. Toggle this behavior by setting the `jruby-puppet.multithreaded` to `true`.
```

Comment:
Would you mind adding a note here along the lines of the following:
>In this mode, the server's memory footprint will be significantly lighter due to no longer needing to run multiple JRuby instances.

Want to make the upside of the new mode clear :)

Reply:
Makes sense to me! Added.